moninag/moninag/urls.py

```diff
@@ -20,6 +20,7 @@
 urlpatterns = [
     url(r'^', include('home.urls')),
     url(r'^api/1/service/', include('service.urls')),
+    url(r'^api/1/check/', include('check.urls')),
     # Registration & Auth
     url(r'^accounts/', include('registration.urls'))
 ]
```

Comment:
The root of problem is that your branch is not up-to-date with dev.

Rebase or merge with dev :upside_down_face: